deploy/helm/charts/templates/lvm-node-service.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.lvmPlugin.metricsListenAddress }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "lvmlocalpv.fullname" . }}-node-service
+  labels:
+    {{- include "lvmlocalpv.lvmNode.labels" . | nindent 4 }}
+spec:
+  ports:
+    - name: metrics
+      port: {{ .Values.lvmPlugin.metricsListenAddress }}
+      targetPort: {{ .Values.lvmPlugin.metricsListenAddress }}
+  selector:
+    {{- with .Values.lvmNode.podLabels }}
+    {{ toYaml . }}
+    {{- end }}
+  type: ClusterIP
```

Comment:
q_: how does the cluster-ip work in the case of Daemonsets?

Reply:
It works in a similar way as that of pods. 
```
Name:	openebs-monitoring-prometheus-lvm-exporter.kube-system.svc.cluster.local
Address: 10.107.162.98
```

However, in this case, we don't need clusterIP. We can simply create a headless service by setting clusterIP as None. This will simply return a list of all the pod IPs.
```
Name:	openebs-monitoring-prometheus-lvm-exporter.kube-system.svc.cluster.local
Address: 10.62.10.114
Name:	openebs-monitoring-prometheus-lvm-exporter.kube-system.svc.cluster.local
Address: 10.62.10.113
```

Reply:
OK. let us make this headless service. Is this similar to node exporter?

Reply:
No, node exporter creates service of type clusterIP.
But for all others(kube-proxy, kubelet, and all), kube-prometheus-stack creates headless service.